src/compiler/app-core/bundle-app-core.ts

```diff
@@ -9,7 +9,7 @@ import { stencilBuildConditionalsPlugin } from '../rollup-plugins/stencil-build-
 import { stencilClientPlugin } from '../rollup-plugins/stencil-client';
 import { loaderPlugin } from '../rollup-plugins/loader';
 import { stencilExternalRuntimePlugin } from '../rollup-plugins/stencil-external-runtime';
-
+import { imagePlugin } from '../rollup-plugins/image-plugin';
```

Comment:
is there a reason why not using the rollup plugin: https://github.com/rollup/plugins/tree/master/packages/image